dotcom-rendering/src/web/components/DynamicPackage.tsx

```diff
@@ -97,79 +168,448 @@ export const DynamicPackage = ({
-						imagePositionOnMobile="bottom"
-						imageSize="large"
-						supportingContent={primary.supportingContent}
-						isDynamo={true}
-					/>
-				</LI>
-				<LI
-					showDivider={true}
-					showTopMarginWhenStacked={true}
-					percentage="25%"
-				>
-					<UL direction="column">
-						{remaining.map((card, cardIndex) => {
-							return (
-								<LI
-									key={card.url}
-									padSides={true}
-									showTopMarginWhenStacked={false}
-									padBottom={
-										// No bottom margin on the last card
-										cardIndex < remaining.length - 1
+			</LI>
+			<LI
+				showDivider={true}
+				showTopMarginWhenStacked={true}
+				percentage="25%"
+			>
+				<UL direction="column">
+					{remaining.map((card, cardIndex) => {
+						return (
+							<LI
+								key={card.url}
+								padSides={true}
+								showTopMarginWhenStacked={false}
+								padBottom={
+									// No bottom margin on the last card
+									cardIndex < remaining.length - 1
+								}
+								padBottomOnMobile={false}
+							>
+								<FrontCard
+									trail={card}
+									containerPalette={containerPalette}
+									containerType="dynamic/package"
+									showAge={showAge}
+									imageUrl={
+										// Always show the image on the first card and only
+										// on the second if there are two items in two
+										cardIndex === 0 ||
+										remaining.length === 2
+											? card.image
+											: undefined
 									}
-									padBottomOnMobile={false}
-								>
-									<FrontCard
-										trail={card}
-										containerPalette={containerPalette}
-										containerType="dynamic/package"
-										showAge={showAge}
-										imageUrl={
-											// Always show the image on the first card and only
-											// on the second if there are two items in two
-											cardIndex === 0 ||
-											remaining.length === 2
-												? card.image
-												: undefined
-										}
-										supportingContent={
-											card.supportingContent
-										}
-									/>
-								</LI>
-							);
-						})}
-					</UL>
-				</LI>
-			</UL>
-		</>
+									supportingContent={card.supportingContent}
+								/>
+							</LI>
+						);
+					})}
+				</UL>
+			</LI>
+		</UL>
 	);
 };
+
+export const DynamicPackage = ({
+	groupedTrails,
+	containerPalette,
+	showAge,
+}: Props) => {
+	let layout:
+		| 'primaryBoosted'
+		| '4orLessStandards'
+		| '5Standards'
+		| '6Standards'
+		| '7Standards'
+		| '8Standards'
+		| '9Standards';
+
+	const snaps = [...groupedTrails.snap].slice(0, 1);
+
+	// We support up to 9 cards
+	const cards = [
+		...groupedTrails.snap.slice(1),
+		...groupedTrails.standard,
+	].slice(0, 9);
+
+	let firstSlice: TrailType[] = [];
+	let secondSlice: TrailType[] = [];
+	let thirdSlice: TrailType[] = [];
+	switch (cards.length) {
+		case 9: {
+			firstSlice = cards.slice(0, 1);
+			secondSlice = cards.slice(1, 5);
+			thirdSlice = cards.slice(5);
+			layout = '9Standards';
```

Comment:
```suggestion
			layout = '9Standards';
			firstSlice = cards.slice(0, 1);
			secondSlice = cards.slice(1, 5);
			thirdSlice = cards.slice(5);
```

It's nice if the layout definition is always the first thing set